nixos/modules/security/ca.nix

```diff
@@ -94,6 +94,14 @@ in
         (Read-only) the path to the final bundle of certificate authorities as a single file.
       '';
     };
+
+    security.pki.cacertPackage = lib.mkOption {
+      type = lib.types.package;
+      readOnly = true;
+      description = ''
+        (Read-only) the path to the final package containing the bundle of certificate authorities as a single file.
+      '';
```

Comment:
maybe add a `defaultText` with the expression from above so that it's clear in the manual what this is?